src/zimcheck/checks.cpp

```diff
@@ -387,7 +387,7 @@ void ArticleChecker::check_item(const zim::Item& item)
         data = item.getData();
 
     if(checks.isEnabled(TestType::REDUNDANT))
-        hash_main[adler32(data)].push_back( item.getIndex() );
+        hash_main[adler32(data)].push_back( {item.getIndex(), item.getClusterIndex(), item.getBlobIndex()} );
 
```

Comment:
Why storing the cluster and blob indices with the entry index (and increasing the memory usage by zimcheck) if that information can be obtain in `ArticleChecker::detect_redundant_articles()`?

Reply:
It was to avoid getting the dirent again in `detect_redundant_articles()`.
But, indeed, it may not worth it. I have changed that.